apps/ledger-live-desktop/src/renderer/modals/AddAccounts/steps/StepChooseCurrency.tsx

```diff
@@ -63,6 +66,7 @@ const StepChooseCurrency = ({ currency, setCurrency }: StepProps) => {
   const base = useFeature("currencyBase");
   const baseGoerli = useFeature("currencyBaseGoerli");
   const klaytn = useFeature("currencyKlaytn");
+  const mock = useEnv("MOCK");
```

Comment:
It seems we want to limit as much as possible the use of the env var directly in the app's code, see https://ledger.slack.com/archives/C0308JT8WS2/p1692373491532989?thread_ts=1691507827.835819&cid=C0308JT8WS2

But there's a debug tool available in the app to set it anyway.